### PR TITLE
The Beast Awakens (calling his flock home)

### PR DESCRIPTION
**Pull request by NPC1314**
## About The Pull Request

Dendor. 

I think savage, nature, forests. Not stone temples, shiny paladins. So, Dendor is moving out, his disciples no longer living in town, embracing their wild side.

dendorr

* New pilgrim class Briar.
Open to all but Aasimar and Dark Elves. Will force set Dendor as patron so won't miss out due to forgetting to set that.
This is a remodeled acolyte in the wids, a pilgrim class with spells and outdoors skills,. Can fill in as a butcher or farmer, generally speaking. Gets unique Briar thorns headwear, pricks non-dendorites who pick it up. Also get the shillelagh, a sturdy club for one or two handed use. 

* Dendor top tier spell added. Its the Kneestinger summoning one, modified to just spawn a single one on the casters location.

* Kneestingers get correct pathing, no longer glowshrooms. Also updated the code, instead of just being a follower of Dendor making you immune, the Trait STINGER IMMUNE is used for that, as it was designed

* Unique recipes for the Briar, they can make shrines to Dendor (their version of Pantheon crossses), the recipe is based on the normal one but uses thorns instead of stakes and needs to be topped with either a saiga head or a volf head, depending on what side of Dendor youre into

*Adds saiga head. Perfect butcher result for saiga

* Dendor paladins are no more, with nature out of the Temple no templar is needed, his followers should be wilder, feral, not so prim and proper like paladins anyways. Instead the Dendor cleric gets a little boost

* Clerics, bit of diversity to their  neck wear depending on their patrons themes, but most important Dendor cleric now gets a dendor helmet, fur cloak instead of tabard, and loses his money

* Acolytes of Dendor no longer allowed

*Some minor cleanup from hydro I stumbled over and a couple minor tweaks to butcheries, neither very important.

Bunch of stuff added, two new spells, two types of shrines they can build to dedcate themselves to either more predatory combat style spells or more growing/foraging etc

*Fixed a llittle jump in the combat music track and did the delag procedure on the combat tracks staring with D. Its export quality 1 in audiopro, couple tracks already had 2 but most got 5. Much smaller size = much less lag when combat mode entered, its hard to detect any audible difference so its a good idea.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [ ] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/cleric.dm [hunk 1]
```diff
@@ -24,41 +24,50 @@
...
 		if("Astrata")
 			wrists = /obj/item/clothing/neck/roguetown/psycross/silver/astrata
 			cloak = /obj/item/clothing/cloak/stabard/templar/astrata
-		if("Dendor")
+			neck = /obj/item/clothing/neck/roguetown/chaincoif
+		if("Dendor")	// good helmet but no money
+			head = /obj/item/clothing/head/roguetown/helmet/heavy/dendorhelm
+			neck = /obj/item/clothing/neck/roguetown/coif
 			wrists = /obj/item/clothing/neck/roguetown/psycross/silver/dendor
-			cloak = /obj/item/clothing/cloak/stabard/templar/dendor
```
Comment (sanshoom) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/cleric.dm:33]: Why remove their tabard? You can put the furcloak on one of their backslots to allow both to pre-exist, can't you?

**Comment by sanshoom** [comment]
This all looks fine, but I don't generally understand the point of removing Paladins from being able to choose Dendor because they're 'prim and proper', it honestly just seems like a stupid reason.

File: code/datums/gods/patrons/divine_pantheon.dm [hunk 2]
```diff
@@ -38,7 +38,7 @@
 	t0 = /obj/effect/proc_holder/spell/invoked/lesser_heal
 	t1 = /obj/effect/proc_holder/spell/targeted/blesscrop
 	t2 = /obj/effect/proc_holder/spell/targeted/beasttame
-	t3 = null
+	t3 = /obj/effect/proc_holder/spell/targeted/conjure_glowshroom // kneestingers
```
Comment (BadAtThisGame302) [inline code/datums/gods/patrons/divine_pantheon.dm:41]: You could probably rename the title of the spell like you renamed everything else from glowshroom to kneestinger(s)

**Comment by NPC1314** [comment]
Either you go for a theme or you make some sort of mediocre halfmeasure.
Dendor lore is a beastly force of nature, that clashes thematically with knights buying expensive finely woven tabards and industrial steel by the boatload.

A God that lives in the wild doesn't need or want knightly orders, if without a Temple it needs no Templars.
Thats my take on Dendor, thats how to make the pantheon matter more, be more than a label on stat variable.
its making lore happen ingame, simply put.

(also the dendor helmet got slapped on its cleric)

**Comment by assortedbeads** [comment]
Let dark elfs be brair folk I dont think nature cares what race you are

**Comment by ThePainkiller** [comment]
Instamerge just because of the shillelagh being added tbh

This is a step towards a very based direction and I love it.

**Comment by NPC1314** [comment]
I'm working on a cooler spell than crops and some totem upgrades too but might need to go in another PR, Ill see if I can squeezze in the spell quick like tho, moment

**Comment by NameChangeStonekeep** [comment]
This sounds more like a concept for Blackstone then stonekeep and the grammar is kinda poor and lack of explanation to why this should be in. so I don't think this will fit at all much

**Comment by Mogelix** [comment]
Welcome this as a churchie.

**Comment by NameChangeStonekeep** [comment]
Still a big no to me does not fit tbh

**Review by Optimism333 — COMMENTED** [review]
If briar is intended to be a wilderness playthrough role, could probably use at least weak (1) skill level in the following three:

- Cooking
- Sewing
- Crafting

That's been the feedback I've heard so far.

**Comment by guayitoo** [comment]
If needed, I can help with rewriting the messages to fit the lore.

Cool stuff, excited to see the fixes Optimism asked for tho.

**Comment by Dagonlives** [comment]
I just tried this role

Some feedback

No sewing, crafting or cooking skill actually makes this super hard to survive with

second note:

We spent 2 hours trying to get a volfhead from over 12+ volf corpses and never got a single one. Either this feature is bugged, or the drop rate is far far too rare.

**Comment by ThePainkiller** [comment]
The volf head is obtained by perfect butchering, meaning you require +4 butchering to get it

**Comment by NPC1314** [comment]
The Briar has butcher 4 so was 30% to get it but its a bit low. Bumped perfect result at 4 to 50%

Set slots to 4 for testing, can be lowered to 2 when testing voer.

Changes to skills 
cook, sew, craft lvl 1 added. Butcher raised to 5, kinda encroaches on butchers thing but lets see how it goes, might not be bad.
blessings after the first no longer are unusable, instead they give +100 devotion if Dendorite  before dissipating.  Takes as much time to collect and make sacrifices as just clicking the pray button and waiting anyways so unlikely to cause issues.

**Comment by NPC1314** [comment]
> If needed, I can help with rewriting the messages to fit the lore.
> 
> Cool stuff, excited to see the fixes Optimism asked for tho.

Feel free, only lore issue I know of is people saying Trolsl are Dendor and some saying they´re Graggarite but maybe there are others, I dont know. If you got improvements to them in other ways thats fine too, fire away.

If Trolls aren't dendors boys the spell does need renaming, maybe Ogre instead of Troll.

**Comment by YuiY1997** [comment]
I can't believe people are still adding the incredibly janky knee-stinger summoning spell I made for blackstone

**Review by Optimism333 — COMMENTED** [review]
Conflict with the merge of #995 due to this pr also touching the desert rider ogg.

**Comment by NPC1314** [comment]
> Conflict with the merge of #995 due to this pr also touching the desert rider ogg.

Sorted

**Review by Optimism333 — APPROVED** [review]

**Pull request merged**